org.wfview.wfview.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: org.wfview.wfview
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
```

Comment:
why not using the 6.6 runtime?

Reply:
Good question. I will try updating it. :-)

Reply:
We've had wfview building with qt6 on windows for some time, it should be alright with Linux as well.

Reply:
Done. Let's see if it builds. :-)

Reply:
It sadly does not build with the 6.6 runtime:


```
wfmain.cpp:2617:40: error: conversion from ‘long int’ to ‘const QVariant’ is ambiguous
 2617 |             if (settings->value("Icon",NULL) != NULL) {
      |                                        ^~~~

wfmain.cpp:2618:63: error: conversion from ‘long int’ to ‘const QVariant’ is ambiguous
 2618 |                 butt.icon = new QImage(settings->value("Icon",NULL).value<QImage>());
      |                                                               ^~~~

...

wfmain.cpp:2618:81: error: expected primary-expression before ‘>’ token
 2618 |                 butt.icon = new QImage(settings->value("Icon",NULL).value<QImage>());
      |                                                                                 ^
wfmain.cpp:2618:83: error: expected primary-expression before ‘)’ token
 2618 |                 butt.icon = new QImage(settings->value("Icon",NULL).value<QImage>());
      |                                                                                   ^
```

@eliggett Any idea?